gradle/plugins/settings.gradle.kts

```diff
@@ -1,3 +1,4 @@
+rootProject.name = "plugins"
```

Comment:
This is new to me. How can I get the warning you want to fix here?

Reply:
Clone [this](https://github.com/scijava/pom-scijava/blob/gradlePlatform/pom.xml)

> 11:35:42: Executing ':playground:dependencies'...

Type-safe project accessors is an incubating feature.
Project accessors enabled, but root project name not explicitly set for 'plugins'. Checking out the project in different folders will impact the generated code and implicitly the buildscript classpath, breaking caching.

Reply:
Okay thanks for explaining.

This happens because you use "Type-safe project accessors" in your project and Gradle applies it then to this build because you included it. (I think Gradle should not do that, because we did not activate the feature in this build.)

Can you please remove the change from this PR, as it is unrelated to the new rules and it is not needed if we work on this project standalone.